src/test/java/nova/core/util/ProgressBarTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2017 NOVA, All rights reserved.
+ * This library is free software, licensed under GNU Lesser General Public License version 3
+ *
+ * This file is part of NOVA.
+ *
+ * NOVA is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * NOVA is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with NOVA.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package nova.core.util;
+
+import nova.core.event.bus.Event;
+import nova.core.event.bus.EventBus;
+import nova.testutils.mod.TestMod;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ *
+ * @author ExE Boss
+ */
+public class ProgressBarTest {
+
+	EventBus<Event> events;
+
+    @Before
+    public void setUp() {
+		this.events = new EventBus<>();
+	}
+
+	@Test
+	public void testProgressBar() {
+		events.on(ProgressBarEvent.class).bind(evt -> System.out.println(evt.message));
```

Comment:
This test should add `evt.message` to a List and verify the results with assertions.